configs/templates/hpc-stack-dev/spack.yaml

```diff
@@ -23,7 +23,7 @@ spack:
       version: [1.78.0]
       variants: ~atomic +chrono +date_time +exception +filesystem ~graph ~iostreams ~locale ~log ~math ~mpi ~numpy +pic +program_options +python ~random +regex +serialization ~signals +system +test +thread +timer ~wave cxxstd=14 visibility=hidden
     bufr:
-      version: [11.7.0]
+      version: [11.7.1]
```

Comment:
Sorry but this may sound confusing. The `hpc-stack-dev` template is supposed to stay in sync with what hpc-stack is currently using until the transition is made at EMC from hpc-stack to spack-stack. Can I ask you to revert this particular change, please?

Reply:
No problem - I reverted this change.